.storybook/addons/theme-switcher-addon/index.js

```diff
@@ -26,8 +26,10 @@ class ThemeWrapper extends React.Component {
   };
 
   render() {
+    // These are dummy breakpoints for testing
+    const breakpoints = ['40em', '52em', '64em', '80em'];
```

Comment:
reminder to remove these

Reply:
Ooooh wait, the logic here is wrong https://github.com/twilio-labs/paste/blob/master/packages/paste-theme/src/themeProvider.tsx#L47

`breakpoints` is on the theme object already, I just did a dumb. https://github.com/twilio-labs/paste/blob/master/packages/paste-theme-tokens/src/default/index.ts#L25


I don't think it was working for another reason that got fixed.

Reply:
Wait, is it wrong? Maybe it's not...

Anyway, regardless we don't need them here

Reply:
Ok I'll remove it

Reply:
Oh the issue was because initially we were passing `customBreakpoints={[]}` which set breakpoints to `[]`.  I removed the prop altogether so we will now get theme breakpoints.